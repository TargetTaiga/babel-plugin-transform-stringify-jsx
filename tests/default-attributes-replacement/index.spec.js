const fs = require('fs');
const path = require('path');
const babel = require('@babel/core');
const plugin = require('../..');

it('should replace default JSX attributes', () => {
    const { code } = babel.transform(fs.readFileSync(path.join(__dirname, 'input.js')).toString(),
        { plugins: [plugin], parserOpts: { plugins: ['jsx']} });
    expect(code).toMatchSnapshot();
});