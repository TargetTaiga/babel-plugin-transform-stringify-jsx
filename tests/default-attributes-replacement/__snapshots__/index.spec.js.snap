// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should replace default JSX attributes 1`] = `"const div = \`<div><label for=\\"button\\">Button</label><button id=\\"button\\" class=\\"myButton\\">Click me</button></div>\`;"`;
